kubernetes_env/kube_env.py

```diff
@@ -478,6 +540,13 @@ def main(args):
                         help="Just do a deployment. "
                         "This means installing bookinfo and Kubernetes."
                         " Do not run any experiments.")
+    parser.add_argument("-so",
```

Comment:
I would use parameters here instead of adding a command per app. That does not scale well. Just use choices and then an if statement for the deploy and remove calls.

Reply:
So still have a separate flags, but just have an umbrella "start app" function?

Reply:
Basically like the --platform argument. Use choices for the apps and then an differentiate in the code.